content/en/docs/ops/integrations/skywalking/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Apache SkyWalking
+description: How to integrate with Apache SkyWalking.
+weight: 32
+keywords: [integration,skywalking,tracing]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+[Apache SkyWalking](http://skywalking.apache.org) is an application performance monitor (APM) system, especially designed for
+microservices, cloud native and container-based architectures.
+
+## Installation
+
+### Option 1: Quick start
+
+Istio provides a basic sample installation to quickly get SkyWalking up and running:
+
+{{< text bash >}}
+$ kubectl apply -f {{< github_file >}}/samples/addons/extras/skywalking.yaml
+{{< /text >}}
+
+This will deploy SkyWalking into your cluster. This is intended for demonstration only, and is not tuned for performance or security.
+
+Istio proxies by default don't send traces to SkyWalking, you will also need to enable the SkyWalking tracing extension provider with
+the following options at installation time:
+
+{{< text bash >}}
+$ --set meshConfig.extensionProviders[0].name=envoy.tracers.skywalking \
```

Comment:
perfer to give a iop segment like:
```yaml
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
spec:
  meshConfig:
    extensionProviders:
      - skywalking:
          service: tracing.istio-system.svc.cluster.local
          port: 11800
        name: skywalking
```

BTW, alse need a mesh-wide telemetry or set defaultProviders

Reply:
`defaultProviders` is set in the next few lines

`--set meshConfig.defaultProviders.tracing[0]=envoy.tracers.skywalking`

Reply:
Modified to IOP segment